src/commands/node/tasks.ts

```diff
@@ -1079,6 +1079,9 @@ export class NodeCommandTasks {
           config.deployment,
         );
         for (const networkNodeServices of config.serviceMap.values()) {
+          if (networkNodeServices.accountId === constants.IGNORED_NODE_ACCOUNT_ID) {
+            continue;
+          }
```

Comment:
Need this filter logic here, otherwise, serviceMap contains deleted nodes,
and nodeClient would fail to ping the no longer running node